src/pybind/mgr/nfs/cluster.py

```diff
@@ -1,3 +1,4 @@
+import errno
```

Comment:
Any reason for importing `errno`?

Reply:
had some experimentation, forgot to remove it.

Reply:
removed. thanks.